src/flb_http_client.c

```diff
@@ -447,6 +447,19 @@ static int process_data(struct flb_http_client *c)
     return FLB_HTTP_MORE;
 }
 
+#if defined FLB_HAVE_TESTS_OSSFUZZ
+int fuzz_process_data(struct flb_http_client *c);
+int fuzz_process_data(struct flb_http_client *c) {
```

Comment:
what about making this function static inline to avoid the extra declaration ?

Reply:
The point of this is to avoid static and its internal linkage?

Reply:
ah got it.